### ISSUE TITLE
Dependency on PCNTL extension

### ISSUE THREAD

**Issue by MorrisJobke** [owncloud/updater#252]
I got this when opening the updater. It seems that it requires an extension to work properly. This should be checked before running the updater.

```
UnexpectedValueException: Could not parse a response for config:list --private. Please check if the current shell user can run occ command. 
Raw output: The process control (PCNTL) extensions are required in case you want to interrupt long running commands - see http://php.net/manual/en/book.pcntl.php HERE IS THE ACTAUL OUTPUT OF THE COMMAND
```

cc @karlitschek @cmonteroluque @DeepDiver1975 and for the other dependency checks @LukasReschke 

We maybe also want to check this on all supported platforms.

**Comment by LukasReschke** [comment]
Caused by https://github.com/owncloud/core/commit/79d2f3186cad9d2412166c9bd414d73dd97cb3d3, doesn't seem like an hard dependency for me.

Probably https://github.com/owncloud/core/pull/22218 should hide this error as well.

**Comment by MorrisJobke** [comment]
It seems that it installed in my 5.6:

<img width="938" alt="bildschirmfoto 2016-02-25 um 09 55 20" src="https://cloud.githubusercontent.com/assets/245432/13314598/5c494d1c-dba6-11e5-8e6e-c9204e12a82d.png">


But php on the command line is a PHP 7

So maybe just an issue with PHP7.

Checking PHP7 also shows it is enabled:

<img width="1438" alt="bildschirmfoto 2016-02-25 um 09 58 17" src="https://cloud.githubusercontent.com/assets/245432/13314611/694f6b5e-dba6-11e5-87a8-81c11a6e1e53.png">

**Comment by MorrisJobke** [comment]
Ah .... thanks @LukasReschke for the pointer. Let me check if I could enhance the `--no-warning` option.

**Comment by MorrisJobke** [comment]
Fix is in https://github.com/owncloud/core/pull/22639

**Comment by karlitschek** [comment]
Great. I would recommend to fix this in 9.0.0
